### DISCUSSION TITLE
posts/531cccb5

### DISCUSSION THREAD

**Discussion by giscus** [ZerolAcqua/zerolacqua.github.io#10] [Announcements]
# posts/531cccb5

嘛，就是一些关于入坑以来玩俄罗斯方块的一些感受吧。

http://zerolacqua.top/posts/531cccb5.html

**Comment by farteryhr** [comment]
还行，非定式人表示，但是实际上io规则有点绕回来了，面对定式人，开局就算freestyle消四/T2，送的4行反而被高级接近PC（如果你垃圾行先过去，就是半PC，“垃圾行在哪都是消四预备的”）的定式放大反击（尤其是io挖掘连中消四伤害爆炸），这时候得开局消二消三甚至2w制造散垃圾才能拒绝圈套，接下来拼实力【

**Reply by farteryhr** [reply]
另个人建议捐赠统一本地化译名架桥（直译实在是不知所云

**Reply by ZerolAcqua** [reply]
woc，大佬来了！感谢大佬对我这点小心得的纠正和补充，十分受用！作为非定式人，我也确实经常吃这种亏。而且在开局时，由于打不出太多火力，总喜欢等对方的火力先过来再打回去，结果就是被对方完整的一套送走，即使没死，也被顶到很高，没来得及挖到长列就只能等死了。再加上经常被手速压制，输给定式人很多都是输在了开局（
